pubspec.yaml

```diff
@@ -4,14 +4,14 @@ version: 1.2.5+2
 homepage: https://github.com/lukepighetti/hydrated
 
 environment:
-  sdk: ">=2.6.0 <3.0.0"
+  sdk: ">=2.5.0 <3.0.0"
 
 dependencies:
   flutter:
     sdk: flutter
 
-  rxdart: ^0.24.1
-  shared_preferences: ^0.5.7+3
+  rxdart: ^0.22.0
```

Comment:
Why are you reverting `rxdart` version? It was me myself that requested and made the PR to upgrade it to the latest version.

People using the latest `rxdart` won't be able to use the package.

Reply:
Refer to #14.

Reply:
This branch is not meant to be merged into master and released. I'll delete it shortly.
We were using it for an internal project that required rxdart 0.22.0.

Reply:
I see, sorry.